services/value-store/projects/src/lib/SaveStream.js

```diff
@@ -14,19 +14,16 @@ export default class SaveStream extends stream.Writable {
     // Manually calculate the ID
     // md5 hash of computed_key + project_id
     const { computed_key: computedKey, project_id: projectId } = chunk;
+
+    const body = Object.assign({}, chunk, {
+      enriched: false,
```

Comment:
👍

Reply:
I actually removed this change as I applied it wrongly on the project and not the location. Then, I decided to not add a false flag at all, as when needed, flag becomes `true` and it's enough to check for truthy values where needed.

Reply:
fine for me